pkg/registry/experimental/controller/etcd/etcd.go

```diff
@@ -105,10 +92,17 @@ func (r *ScaleREST) Update(ctx api.Context, obj runtime.Object) (runtime.Object,
 	if err != nil {
```

Comment:
we're not doing anything with the ResourceVersion in the scale object... are we expecting a mismatch there to produce a resource conflict here?